frontend/src/utils/getTypedDrawObject/getTypedDrawObject.ts

```
@@ -0,0 +1,92 @@
+
+export function getTypedDrawObject(
+  str: string,
+  roughCanvas: RoughCanvas,
+  ctx: CanvasRenderingContext2D,
+): BaseObject | null {
+  const parsedString: BaseObject = JSON.parse(str);
+  const type: string = parsedString.drawType;
+
+  switch (type) {
+    case 'rectangle':
+      const r = new RectangleObject(parsedString);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)